snippets/ztp-patch-argocd-hub-cluster.adoc

```diff
@@ -1,5 +1,70 @@
 :_mod-docs-content-type: SNIPPET
-. To install the {ztp} plugin, patch the ArgoCD instance in the hub cluster by using the patch file that you previously extracted into the `out/argocd/deployment/` directory.
+. To install the {ztp} plugin, patch the ArgoCD instance in the hub cluster with the relevant multicluster engine (MCE) subscription image.
+Customize the patch file that you previously extracted into the `out/argocd/deployment/` directory for your environment.
+
+.. Select the `multicluster-operators-subscription` image to work with your {rh-rhacm} version.
++
+--
+.multicluster-operators-subscription image versions
+[cols="1,1,1,1,3", options="header"]
+|====
+|{product-title} version
+|{rh-rhacm} version
+|MCE version
+|MCE RHEL version
+|MCE image
+
+|4.14, 4.15, 4.16
+|2.8, 2.9
+|2.8, 2.9
+|RHEL 8
+|`registry.redhat.io/rhacm2/multicluster-operators-subscription-rhel8:v2.8`
```

Comment:
just a nit, but I think having the second URL rendered on another line would improve scanability.